### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by d2weber**
## Description of change

Please write a summary of your changes and why you made them. 

Be sure to reference any related issues by adding `closes issue #`.

## How Has This Been Tested (if applicable)?

Please describe the tests that you ran to verify your changes.

### CODE REVIEW

**Review by oddgrd — COMMENTED** [review]
Hey and thanks for this! I think we can take this one step further and remove the `cargo shuttle project status` command, project new will no longer return before the project is ready. But, we should also add a comment after `cargo shuttle project new` since the project might have been created in the init flow above. Something like `# if you didn't create your environment on shuttle already`.

**Comment by d2weber** [comment]
Yes I like that idea. I also mentioned that steps should be performed within the project directory as it makes the example more complete.

**Review by oddgrd — APPROVED** [review]
Thanks! This is perfect, but I think we should hold off on merging in until we merge #671, since currently if you pass a path that doesn't exist to the init command it will just error.

**Pull request merged**
